testdata/project-v2/Makefile

```diff
@@ -54,7 +54,8 @@ generate: controller-gen
 docker-build: test
 	docker build . -t ${IMG}
 	@echo "updating kustomize image patch file for manager resource"
-	sed -i'' -e 's@image: .*@image: '"${IMG}"'@' ./config/default/manager_image_patch.yaml
+	kustomize edit set image $IMG ./config/default/manager_image_patch.yaml
```

Comment:
You need to do the following or something better.

- cd config/manager
- kustomize edit set image controller=${IMG}
- cd ../..

Reply:
changed it as per above